src/jobtypes.py

```diff
@@ -68,11 +67,27 @@ def __init__(self, repo_name='mozilla-inbound'):
                 #  opt test web-platform-tests-4"
                 if job['build_system_type'] == 'buildbot':
                     testtype = job['ref_data_name'].split(' ')[-1]
+                    job_identifier = job['ref_data_name']
 
                 # taskcluster's testtype is a part of its 'job_type_name' like reftest-2
                 # for [TC] Linux64 reftest-2
                 else:
-                    testtype = job['job_type_name'].split(' ')[-1]
+                    # we don't want job of build kind in seta
+                    if job['job_type_symbol'] == "I":
+                        continue
+                    else:
+                        # It's a bit of hard code actually, because the test name on taskcluster
+                        # comes to a sort of combination(e.g desktop-test-linux64/debug-jittests-3)
+                        # and asan job can been referenced as a opt job. BTW, job like
+                        # 'MacOSX64 Static Analysis Opt' will been leave aside because
+                        # they are not test job anyway
+
+                        separator = job['platform_option'] \
+                            if job['platform_option'] != 'asan' else 'opt'
+
+                        testtype = job['job_type_name'].split(
+                            '{buildtype}-'.format(buildtype=separator))[-1]
+                        job_identifier = job['ref_data_name']
```

Comment:
can you give a couple examples of what data comes in and what we put in separator/testtype/job_identifier?